.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- 
+Please make sure you have read and understood the contributing guidelines;
+https://github.com/openkruise/kruise/blob/master/CONTRIBUTING.md -->
+
+### Ⅰ. Describe what this PR did
+
+
+### Ⅱ. Does this pull request fix one issue?
+<!--If that, add "fixes #xxxx" below in the next line, for example, fixes #15. Otherwise, add "NONE" -->
+
+
+### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
```

Comment:
can you remove the Chinese explanation? thanks.

Reply:
How about this?

List the added test cases (unit test/integration test) if any, please explain if no tests are needed.